subworkflows/local/normalize/tests/main.nf.test.snap

```diff
@@ -0,0 +1,20 @@
+{
+    "test profile": {
+        "content": [
+            [
+                "202411141755_normalized_counts.tsv:md5,a0124c7a24bd04296f441d9ade82a05f"
```

Comment:
it seems to append the date to the name, which may then make it change in different runs

Reply:
can we remove adding the date from the prefix in the module itself? (since it is an R script)

Reply:
I see that someone else also opened an issue about this: https://github.com/nf-core/nanostring/issues/69

removing the timestamps would help in these tests

Reply:
Nice catch, since we have to wait on this PR regardless, I'll tackle that issue before merging this one then.